app/stores/AppStore.js

```diff
@@ -96,21 +109,23 @@ export default class AppStore extends Store {
     this.getTermsOfUseAcceptanceRequest.execute();
   };
 
+  _getSendLogsChoice = async () => {
+    const sendLogs = await this.getSendLogsChoiceRequest.execute().promise;
+    ipcRenderer.send('send-logs-choice', sendLogs);
```

Comment:
It is because _getSendLogsChoice is called during AppStore setup and a moment when we actually read current user's send-logs-choice value that we need to send to the main process. I'm open to suggestions :)